ts/navigation/WalletNavigator.ts

```diff
@@ -1,11 +1,15 @@
 import { StackNavigator } from "react-navigation";
 import { AddPaymentMethodScreen } from "../screens/wallet/AddPaymentMethodScreen";
 import { CreditCardsScreen } from "../screens/wallet/CreditCardsScreen";
+import { ManuallyIdentifyTransactionScreen } from "../screens/wallet/ManuallyIdentifyTransactionScreen";
 import { TransactionDetailsScreen } from "../screens/wallet/TransactionDetailsScreen";
 import { TransactionsScreen } from "../screens/wallet/TransactionsScreen";
 import { WalletHomeScreen } from "../screens/wallet/WalletHomeScreen";
 import ROUTES from "./routes";
 
+/**
+ * TODO: migrate WALLET_TRANSACTION_SUMMARY on a new navigator for screens which does not visualize the footer menu.
```

Comment:
Should the tab navigator really be hidden during payments? It would appear so based on the [invision](https://projects.invisionapp.com/share/48GKAY7S9AZ#/screens/291923319), but I wouldn't dismiss the option of having the tab navigator everywhere without even considering it 

cc @cloudify

Reply:
cc @matteodesanti

Reply:
I don't think so

Reply:
cc @spizzichi

Reply:
@valentinacala the amount value should not be necessary. I know that it is provided by the QR code, but i don't think is mandatory. Please check and if i'm right please remove it.

@fgiobergia the tab navigator should only be hidden during a multi-step flows.. In this specific case i think we should simply add a "cancel" button, that take back to the wallet main screen. The back button in the topbar take back to the previous screen (camera-mode).
The "continue" button should be disabled by default.  
I've updated the mockup here https://invis.io/48GKAY7S9AZ#/291923319_2-3-1_-_Inserisci_Dati_Avviso

<img width="347" alt="manually_insert_notice_data" src="https://user-images.githubusercontent.com/8171284/41103766-36e9c0c4-6a6a-11e8-96d6-6fc8d14f7658.png">

Reply:
Thank you!
@valentinacala it probably makes sense to have a separate navigator handle this payment process. When the "cancel" button is pressed, we can then return to the wallet home page

Reply:
@matteodesanti checking the data provided by the remote validation and activation processes, there is always only one amount reported, the actual one. 
Maintaining the field Amount it will be possible to compare the amount read by the user on the notice and the current amount, alerting him if the amount changed in the meanwhile. It is the lone approach I identified to obtain the following screen being displayed when:
<img width="212" alt="updated payment" src="https://user-images.githubusercontent.com/38431762/41234808-8a35d882-6d8d-11e8-8b29-038773ef9cf7.png">

Reply:
It makes sense.. basically those are metadata attached to the message or returned by the QR code, so I think we can easy compare them with the amount provided by pagoPA.
In the meanwhile i've slightly modified the screen: it should look like this  https://invis.io/48GKAY7S9AZ#/302489575_1-4a_-_Riepilogo_Pagamento 


![payment_recap](https://user-images.githubusercontent.com/8171284/41469259-d5864142-70ac-11e8-88c7-5475324be357.png)